core/build.gradle

```diff
@@ -106,7 +106,7 @@ dependencies {
     optionalApi 'io.prometheus:simpleclient_common'
 
     // Netty
-    [ 'netty-transport', 'netty-codec-http2', 'netty-codec-haproxy', 'netty-resolver-dns' ].each {
+    [ 'netty-transport', 'netty-codec-http2', 'netty-codec-haproxy', 'netty-resolver-dns', 'netty-handler-proxy' ].each {
```

Comment:
`netty-handler-proxy` could be declared at `implementation` configuration? I know we need to move some other netty modules from `api` to `implementation`, but I thought this may be a good baby step.